amqphandler/amqphandler.go

```diff
@@ -60,6 +60,8 @@ const (
 type AmqpHandler struct { //nolint:stylecheck
```

Comment:
May be better make maxLenLogMessage configurable?

Reply:
As per @al1img  suggestion I will rename it to `TrimAMQPDataLogs`
I considered this option prior, but this naming might be confusing: only IAM AMQP message is truncated. Users might treat `maxLenLogMessage` as an option to limit all logs size for CM